### DISCUSSION TITLE
NEURON simulations on the fly

### DISCUSSION THREAD

**Discussion by PALMS-gui** [wmglab-duke/ascent#12] [Ideas]
Neuron simulations to generate a response (not threshold finding) for a given fiber (or a small set) don't take much time. 
It would be cool to have a class with some abstract methods allowing such small neuron simulations.
Example use case: "manually" refine activation threshold without creating another experiment

**Comment by thedannymarsh** [comment]
Thanks for the feedback! This would be a really nice feature, which we unfortunately don't have the time to implement currently. A modified submit.py script would definitely get the job done though!
